src/project-ontology.ttl

```diff
@@ -0,0 +1,135 @@
+@prefix : <http://bibsysdev.github.io/src/project-ontology#> .
+@prefix foaf: <http://xmlns.com/foaf/0.1/> .
+@prefix nva: <http://bibsysdev.github.io/src/nva#> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix schema: <https://schema.org/> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+:Project a rdfs:Class ;
+  rdfs:comment "An organised effort designed to achieve a particular goal" ;
+  rdfs:subClassOf schema:Project .
+
+:Identifier a rdfs:Class ;
+  rdfs:comment "An identifier that uniquely identifies something" .
+
+:CristinIdentifier a rdfs:Class ;
+  rdfs:comment "An identifier that uniquely identifies the type of the domain scope in Cristin" ;
+  rdfs:subClassOf :Identifier .
+
+:identifier a rdf:Property ;
+  rdfs:comment "Property relating a project to its identifiers" ;
+  rdfs:domain :Project ;
+  rdfs:range :Identifier .
+
+:value a rdf:Property ;
+  rdfs:comment "A value associated with an Identifier" ;
+  rdfs:domain :Identifier ;
+  rdfs:range xsd:string .
+
+:title a rdf:Property ;
+  rdfs:comment "The name of a thing" ;
+  rdfs:domain :Project, :Grant ;
+  rdfs:range xsd:string .
+
+:language a rdf:Property ;
+  rdfs:comment "The main language of the project" ;
+  rdfs:domain :Project .
+
+:alternativeTitle a rdf:Property ;
+  rdfs:comment "An alternative or translated name for a project" ;
+  rdfs:domain :Project ;
+  rdfs:range rdf:langString .
+
+:startDate a rdf:Property ;
+  rdfs:comment "The point at which something was started" ;
+  rdfs:domain :Project, :Contribution ;
+  rdfs:range xsd:dateTime .
+
+:endDate a rdf:Property ;
+  rdfs:comment "The point at which something was terminated" ;
+  rdfs:domain :Project, :Contribution ;
+  rdfs:range xsd:dateTime .
+
+:grant a rdf:Property ;
+  rdfs:comment "Relates a project to its source of financial support" ;
+  rdfs:domain :Project ;
+  rdfs:range :Grant .
+
+:FundingStream a rdfs:Class ;
+  rdfs:comment "A source of financial support for a project, a funding stream" .
+
+:fundingStream a rdf:Property ;
+  rdfs:comment "Relates a grant to a funding stream" ;
+  rdfs:domain :Grant ;
+  rdfs:range :FundingStream .
+
+:Funder a rdfs:Class ;
+  rdfs:subClassOf nva:Organization .
+
+:Grant a rdfs:Class ;
+  rdfs:comment "A sum of money given to a project to enable the project to meet its goals" ;
+  owl:sameAs <http://bibsysdev.github.io/src/funder-ontology#Project> .
+
+<http://bibsysdev.github.io/src/funder-ontology#Project> a rdfs:Class ;
+  rdfs:comment "We consider funder projects to be synonymous with our grant concept" .
+
+:grantId a rdf:Property ;
```

Comment:
a project_code in Cristin used for fundings is not the same as cristin_project_id. A project_code might look like this:
`"project_code" : "ISTJENESTEN",`